tools/st2ctl

```diff
@@ -51,6 +51,7 @@ function print_usage() {
     echo "usage: st2ctl {reload, clean}"
     echo "optional arguments:"
     echo "  --register-all                Register all."
+    echo "  --register-triggers           Register all triggers."
```

Comment:
+1 to remove this file. It's not used in new packages (only old deprecated system).

Apart of that, we can remove directories like https://github.com/StackStorm/st2/tree/master/st2auth/packaging, since it's part of old deprecated process as well.